app/main.py

```diff
@@ -1,8 +1,51 @@
 class Car:
-    # write your code here
-    pass
+
+    def __init__(self, comfort_class, clean_mark, brand):
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+
+    def __init__(
+            self,
+            distance_from_city_center,
+            clean_power,
+            average_rating,
+            count_of_ratings
+    ):
+
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = round(average_rating, 2)
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, list_cars):
+        income_per_cars = 0
+
+        for i in list_cars:
+            if i.clean_mark < self.clean_power:
+                income_per_cars += self.calculate_washing_price(i)
+                self.wash_single_car(i)
+
+        return round(income_per_cars, 2)
+
+    def calculate_washing_price(self, car):
+        difference_between_cars = self.clean_power - car.clean_mark
+        comfort_minus_difference = car.comfort_class * difference_between_cars
+        price_per_car = comfort_minus_difference * self.average_rating
+
+        return price_per_car / round(self.distance_from_city_center, 1)
+
+    def wash_single_car(self, car):
+        car.clean_mark = self.clean_power
```

Comment:
if I want to just call this method on the object? 
Ex:
ws = CarWashingStation(6, 5, 3.9, 11) # clean_power = 5
ford = Car(2, 7, 'Ford') # clean_mark  7
ford.wash_single_car()
ford.clean_mark == 5 or maybe 7 ?